infra/production/base/main.tf

```diff
@@ -0,0 +1,40 @@
+terraform {
```

Comment:
Define all dependencies needed to run Layer8, InfluxDB, and any new services that may be introduced in the future. This base folder should be moved to a new repository when our other products start using AWS.

In this code, we define:

- Secure and cost-efficient networking solution
- IAM Role to access AWS ECS
- S3 bucket for storing ECS container config
- ECR for storing docker image